app/components/formatted_time.test.js

```diff
@@ -18,70 +18,69 @@ describe('FormattedTime', () => {
     it('should render correctly', () => {
         console.error = jest.fn();
 
-        let wrapper = renderWithIntl(
+        const viewOne = renderWithIntl(
```

Comment:
Should've extracted {rerender} and use it to render again on change in props but it makes it hard to implement when the component under test (`FormattedTime` in this case) is not a parent but child of `IntlProvider`.